src/index.js

```diff
@@ -233,5 +255,6 @@ function getClassName(parseClass) {
 
 module.exports = {
   ParseServer: ParseServer,
-  S3Adapter: S3Adapter
+  S3Adapter: S3Adapter,
```

Comment:
it's 'simpler' in the sense that everything is available from the 1st import, but we don't expose all of them either...